.github/workflows/prune-branches.yml

```
@@ -0,0 +1,31 @@
+      - main # Replace 'main' with your default branch if different
+  schedule:
+    - cron: "0 0 * * 0" # Optional: Run weekly on Sundays at midnight
+
+jobs:
+  prune:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repository
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)